content/chapters/compute/lecture/slides.mdpp

```diff
@@ -18,28 +18,26 @@ revealOptions:
 ## Processes
 
 * A process is a **running program**
-    * It is a collection of resources/abstractions (Memory, CPU, I/O)
+    * It is a collection of resources(Memory, CPU, I/O) or abstractions(VAS, threads, file descriptors) 
 * An application can spawn multiple processes
-* The OS allows each thread to run on a **core** for a few milliseconds (time slice)
-    * See [the corresponding section](#2-threads) for details regarding threads
+* The OS allows each [thread](#2-threads) to run on a **core** for a few milliseconds (time slice)
```

Comment:
```suggestion
* The OS allows each [thread](#2---threads) to run on a **core** for a few milliseconds (time slice)
```

Reply:
I don`t see any difference in functionality, is this intended?

Reply:
My bad. I've updated the section names and now they don't have numbers anymore, so it should be `#threads`. I also updated my comment.